sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/ExpressionCompiler.java

```diff
@@ -999,6 +999,7 @@ node, new DBSPTypeBool(CalciteObject.EMPTY, false), DBSPOpcode.EQ,
             case OTHER_FUNCTION: {
                 String opName = call.op.getName().toLowerCase();
                 switch (opName) {
+                    case "trunc":
```

Comment:
Maybe also fixup documentation to make it clear that TRUNC is a synonym of TRUNCATE, since currently they look like two different functions.

Reply:
that's what the documentation says:
> same as TRUNCATE(value [, digits]) (result is a DECIMAL value).